src/components/pages/Home/Header.module.css

```diff
@@ -1,34 +1,14 @@
-.logoText {
-  margin: var(--spacer) auto;
-  max-width: 80%;
-}
-
-.dataXChange {
-  max-width: 50%;
-}
-
-.dataXChange div[class*='conic'] {
-  width: 100%;
-  height: 100%;
-  background: var(--brand-pink);
-  background-image: conic-gradient(
-    from 180deg,
-    var(--brand-pink),
-    var(--brand-violet)
-  );
+.container {
+  margin-bottom: calc(var(--spacer) * 2);
 }
 
 .tagLine {
-  font-size: var(--font-size-h1);
-  font-weight: var(--font-semibold);
-  color: var(--brand-grey-dimmed);
-  margin-top: calc(var(--spacer) * 4);
-  margin-bottom: calc(var(--spacer) * 2);
-  text-transform: lowercase;
+  font-weight: bold;
+  margin-top: var(--spacer);
+  margin-bottom: var(--spacer);
```

Comment:
I like the proposal by @oceanByte to sort CSS by property type
Maybe we can follow this approach going forward:
Example: https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/

Please add to this if needed @oceanByte